PowerEditor/installer/nativeLang/italian.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version = "1.0" encoding = "utf-8" ?>
 <!--
-    Italian translation for Notepad++ 8.8.0
-    Last modified Sun, Apr 21st, 2025.
+    Italian translation for Notepad++ 8.8.2
```

Comment:
Please don't guess the version number without the official announcement.
Use the current version number instead.